src/sql/parts/modelComponents/viewBase.ts

```diff
@@ -91,4 +96,18 @@ export abstract class ViewBase extends AngularDisposable implements IModelView {
 			// TODO add error handling
 		});
 	}
+
+	registerEvent(componentId: string, eventName: string) {
```

Comment:
You define `eventName` here but never use it. Will you be addressing this in a future update? Or can you remove it?